appendices/spirvenv.txt

```diff
@@ -304,6 +304,21 @@ ifdef::VK_EXT_shader_demote_to_helper_invocation[]
 endif::VK_EXT_shader_demote_to_helper_invocation[]
 |====
 
+[NOTE]
+.Note
+====
+1::
+    <<features-shaderStorageImageExtendedFormats,pname:shaderStorageImageExtendedFormats>>
+    feature is not strictly necessary to be enabled to use
+    code:StorageImageExtendedFormats SPIR-V capability.
+    _Extended storage image formats_ may: support
+    `VK_FORMAT_FEATURE_STORAGE_IMAGE_BIT` regardless of
+    pname:shaderStorageImageExtendedFormats being supported or enabled.
+    But the application must: enable the feature if it relies only on the
+    <<features-required-format-support,required format support>> of the
+    _extended storage image formats_.
```

Comment:
I think the first two sentences are fine but I don't know what you mean by the third.

Reply:
Hm thought I removed that. I will reremove it later. It relates to the confusing part I relay in the "BTW" point in the OP.